workitem/workitem_repository.go

```diff
@@ -314,6 +317,37 @@ func (r *GormWorkItemRepository) List(ctx context.Context, criteria criteria.Exp
 	return res, count, nil
 }
 
+// ListChildren get all child work items
+func (r *GormWorkItemRepository) ListChildren(ctx context.Context, parent string) ([]*app.WorkItem, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "workitem", "children", "query"}, time.Now())
+
+	where := "id in (select work_item_links.target_id from work_item_links left join work_item_link_types on work_item_links.link_type_id = work_item_link_types.id where work_item_link_types.forward_name='relates to' and work_item_links.source_id = ?)"
```

Comment:
Hard coded table name has changed in f44ff7b2fbf2c30c19b945b254c5e1673f83584a and f120c8eae601bc25a32b892ac80128bbc9475338 . Removing left join completed in f44ff7b2fbf2c30c19b945b254c5e1673f83584a